plugins/utils/zellij.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "zellij-nvim";
```

```suggestion
  name = "zellij";
```

This is the name that the nixvim module will have, `zellij` in this case.

---

done